mmedit/models/editors/ggan/ggan.py

```diff
@@ -79,7 +79,7 @@ def train_discriminator(self, inputs: dict,
         Returns:
             Dict[str, Tensor]: A ``dict`` of tensor for logging.
         """
-        real_imgs = inputs['img']
+        real_imgs = torch.stack([data.gt_img for data in data_samples])
```

Comment:
why do we still stack tensor in models' forward?
Have we decided to stack tensors in `data_preprocessor`?

Reply:
We may need check this point for all models